components/brave_rewards/browser/rewards_service.h

```diff
@@ -155,6 +158,12 @@ class RewardsService : public KeyedService {
     const GetPendingContributionsTotalCallback& callback) = 0;
   virtual void GetRewardsMainEnabled(
     const GetRewardsMainEnabledCallback& callback) const = 0;
+  virtual void GetPendingContributions(
+    const GetPendingContributionsCallback& callback) = 0;
+  virtual void RemovePendingContribution(const std::string& publisher_key,
```

Comment:
both the remove methods need callbacks for error handling

Reply:
and what should we do on error?

Reply:
currently we don't have flows for this small errors, like delete was not successful or this action was not successful

Reply:
we don't need error handler as observer return error to the UI